code/src/Restart.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+
+const Restart = () => {
+  const resetSurvey = () => {
+    window.location.reload();
+  }
```

Comment:
You don't need to reload the page, by resetting the state variables into their initial values, the form will go to it's initial state. For example:
`setShowQuestion(false);
  setPlace('');
 setRating('');
  setFeedback('');
 setShowSummary(false);`